telegram/user.py

```diff
@@ -124,28 +132,24 @@ def de_list(cls, data, bot):
     def mention_markdown(self, name=None):
         """
         Args:
-            name (:obj:`str`): If provided, will overwrite the user's name.
-
+            name (:obj:`str`): If provided, will overwrite the user's full name.
         Returns:
             :obj:`str`: The inline mention for the user as markdown.
         """
-        if not name:
-            return util_mention_markdown(self.id, self.name)
-        else:
+        if name:
```

Comment:
This change is out of scope... we're more likely to merge simple self contained pull requests...
IMO this is fine this time though :)

Reply:
Will separate into 2 PRs next time.

Reply:
I changed the link methods so they return None on default and fixed a mistake I spotted in `Message.link`.